### PR TITLE
loginPage

### PR DESCRIPTION
**Pull request by Kelvin-Ofili**

### CODE REVIEW

File: src/components/authContainer/styles.module.scss
Comment (The-Anomaly) [inline src/components/authContainer/styles.module.scss:1]: Why are there changes to this component?

File: src/features/authentication/login/index.tsx [hunk 1]
```diff
@@ -1,10 +1,105 @@
...
+    submit(data);
+  };
+  const { isMobile } = useMobile({});
   return (
     <>
-      <AuthContainer type="side">Login</AuthContainer>
+      {isMobile ? (
+        <UnsupportedScreenSize />
+      ) : (
+        <section>
```
Comment (The-Anomaly) [inline src/features/authentication/login/index.tsx:54]: Why is the auth container wrapped in a section tag? Is the tag performing any specific function?

File: src/features/authentication/login/index.tsx [hunk 2]
```diff
@@ -1,10 +1,103 @@
...
+                  </div>
+                  <p className={`${styles.spread__flex__p}`}>Stay signed in</p>
+                </div>
+                <p className={`${styles.blue} ${styles.spread__p}`}>Forgot password</p>
+              </div>
+              <Button type={"fill"} onClick={handleSubmit(onSubmit)} className={`${styles.btn}`}>
+                Sign in
+              </Button>
+              <p className={`${styles.foot}`}>
+                Don't have an account? <span className={`${styles.blue}`}>Sign up</span>
```
Comment (The-Anomaly) [inline src/features/authentication/login/index.tsx:96]: The "Sign up" text here, should be a <Link> not <span>. And it should link to the signup page

File: src/features/authentication/login/index.tsx [hunk 3]
```diff
@@ -1,10 +1,103 @@
...
+                value={watch("password")}
+              />
+              <div className={`${styles.spread}`}>
+                <div className={`${styles.spread__flex}`}>
+                  <div className={`${styles.spread__flex__check}`}>
+                    <input type="checkbox" />
+                  </div>
+                  <p className={`${styles.spread__flex__p}`}>Stay signed in</p>
+                </div>
+                <p className={`${styles.blue} ${styles.spread__p}`}>Forgot password</p>
```
Comment (The-Anomaly) [inline src/features/authentication/login/index.tsx:90]: Forgot password should be a <Link> that leads to the forgot password page

Comment (Kelvin-Ofili) [reply src/features/authentication/login/index.tsx:90]: fixed. Thanks.

**Pull request merged**